### PR TITLE
fix edit applcation forms layout

### LINKED ISSUE #344: Application edit form update to new form system.
Application edit form was missed in the form refactor

### PR DESCRIPTION
**Pull request by ascobie**
fix #344

A couple of forms were missed in the form refactor to bl-complex-form. Added back in the actionColor and actionName properties that were missed off when simple form implemented a complex form.

Also a couple of CSS tweaks to add some spacing between buttons.

### CODE REVIEW

File: app/styles/base/forms.scss [hunk 1]
```diff
@@ -296,3 +296,7 @@ input.search-input {
     border: none;
     outline: none;
 }
+
+bl-submit-btn {
```
Comment (timotheeguerin) [inline app/styles/base/forms.scss]: can this be a bit more specific that every `bl-submit-btn`

Comment (ascobie) [inline app/styles/base/forms.scss]: nah i think it works fine with the way we do forms. every form has a submit button (or 2) and then a cancel button which is just a <button />. I have yet to see an example of it adding padding where it shouldn't and if i do i will fix it. Though i could nest it under: ".form-buttons" which would narrow it down a little more.

Comment (timotheeguerin) [inline app/styles/base/forms.scss]: it is just here it will be applied to every bl-submit-btn even if it is not in a form

Comment (ascobie) [inline app/styles/base/forms.scss]: nested it under form-buttons selector. this will be better.

Comment (ascobie) [inline app/styles/base/forms.scss]: nested under form-buttons selector.

File: app/components/base/form/form-page/form-page.scss [hunk 2]
```diff
@@ -43,7 +43,7 @@
 
         > .section-content {
             flex: 1;
-            padding: 0 25px;
+            padding: 0 5px;
```
Comment (timotheeguerin) [inline app/components/base/form/form-page/form-page.scss]: I think 5 looks way to compact. Why did you want to change it?

image

**Pull request merged**